### PR TITLE
Readme revamp

### PR DESCRIPTION
**Pull request by GingrPowr**
Revamped README.md:

- Reformated
- Added a simple description (partially copied from command tool description)
- Added a ToC
- Added a simple first steps: install python, install deezer-downloader, first setup, ...
- Renamed some points
- Typos

### CODE REVIEW

**Comment by GingrPowr** [comment]
Issue #114

File: README.md [hunk 1]
```diff
@@ -1,71 +1,95 @@
...
+
+### 4. Set the config file
+Retrieve the template: 
+```bash 
+deezer-downloader -t
+```
+You need to set at least:
+- under `[deezer]`: `cookie_arl`, your **NON PREMIUM** arl key
+- under `[youtubedl]`: `command`, your yt-dlp install path\
+As stated in the config template, deezer-downloader do NOT keep yt-dlp updated, you will have to monitor this yourself.
```
Comment (cyber-g) [inline README.md:61]: I suggest to change the sentence as follows : 

As stated in the config template, deezer-downloader does NOT keep `yt-dlp` updated for you; updating `yt-dlp` is your responsibility.

**Comment by cyber-g** [comment]
Very nice contribution ! My workflow corresponds exactly to work "Get started" section and the proposed documentation is very clear and efficient. 

(I am not a user of the other sections so I cannot really evaluate the section "Specific use cases
")

File: README.md (hunk 1 above)
Comment (GingrPowr) [reply README.md:61]: Yes, if it suits @kmille I'm okay with your modification.

**Comment by GingrPowr** [comment]
@cyber-g Thanks! Same for me, the "Specific use cases" is straight-up copy-pasted from the current readme, I barely touched it (maybe some typos or formatting).

**Comment by kmille** [comment]
Hey,
are you fine when I change things in your MR and squash the commits? I think that's easier than giving feedback to each line.

**Comment by kmille** [comment]
I changed some things. I will read through it again in the next days. Any feedback?

**Pull request merged**